docs/getting_started.md

```
@@ -1,80 +1,109 @@
 Access the Kai settings from `Extensions > Konveyor AI Extension for VS Code settings icon > Settings`.
 
-You can configure basic settings such as:
-- Configure the solution server endpoint 
-- Select the editor view for analysis: diff or merge
-- (Optional) Choose the agent mode for automated analysis with manual review
-- (Optional) Choose the demo mode to use cached responses from the LLM 
+You can configure settings such as:
+
+- The **solution server**. See [MCP Solution Server setup](../kai_mcp_solution_server/README.md)
+- **Agent mode** for automated analysis with manual review
+- **Demo mode** to use cached responses from the LLM. More details can be found [here](demo_mode.md).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)